cves/CVE-2012-0022.yml

```diff
@@ -175,8 +195,12 @@ lessons:
     applies:
     note:
   distrust_input:
-    applies:
-    note:
+    applies: true
+    note: |
+      The input -- in the form of parameters -- from a user has been proven in
```

Comment:
The second sentence is a dependent clause.  Either combine them with a semicolon or add a subject to the second sentence.